packages/live-preview-sdk/src/messages.ts

```diff
@@ -164,6 +166,7 @@ export type MessageFromEditor = (
   | DebugModeEnabledMessage
   | RequestedEntitiesMessage
 ) & {
+  data: any;
```

Comment:
are we cleaning up the typings in another PR then?

Reply:
yes, we need a couple of PRs before work on this repo & the packages one is complete